community/cypher/cypher-logical-plans-3.4/src/main/scala/org/neo4j/cypher/internal/v3_4/logical/plans/ProcedureSignature.scala

```diff
@@ -56,8 +56,14 @@ case class UserFunctionSignature(name: QualifiedName,
 object QualifiedName {
   def apply(unresolved: UnresolvedCall): QualifiedName =
     QualifiedName(unresolved.procedureNamespace.parts, unresolved.procedureName.name)
+
   def apply(unresolved: FunctionInvocation): QualifiedName =
     QualifiedName(unresolved.namespace.parts, unresolved.functionName.name)
+
+  def apply(namespacedName: String): QualifiedName = {
```

Comment:
This new method does not appear to be called by any of the other new code

Reply:
I know, I used it from my old solution and found it very helpful (and thought it might be helpful in the future, too). Do you think we should delete it?

Reply:
If it is unused, I think it should be deleted. It looks easy enough to add back if we find a need in future.